modules/openapi-generator/src/main/resources/go-experimental/model_doc.mustache

```diff
@@ -36,37 +36,40 @@ but it doesn't guarantee that properties required by API are set
 {{#vars}}
 ### Get{{name}}
 
-`func (o *{{classname}}) Get{{name}}() {{dataType}}`
+`func (o *{{classname}}) Get{{name}}() {{vendorExtensions.x-basetype}}`
 
 Get{{name}} returns the {{name}} field if non-nil, zero value otherwise.
 
 ### Get{{name}}Ok
 
-`func (o *{{classname}}) Get{{name}}Ok() ({{dataType}}, bool)`
+`func (o *{{classname}}) Get{{name}}Ok() (*{{vendorExtensions.x-basetype}}, bool)`
 
 Get{{name}}Ok returns a tuple with the {{name}} field if it's non-nil, zero value otherwise
 and a boolean to check if the value has been set.
 
+### Set{{name}}
+
+`func (o *{{classname}}) Set{{name}}(v {{vendorExtensions.x-basetype}})`
+
+Set{{name}} sets {{name}} field to given value.
+
+{{#isNullable}}
 ### Has{{name}}
```

Comment:
`Has...` is defined for every non required types, not just for nullables I believe.